landing-pages/site/content/en/blog/airflow-survey.md

```diff
@@ -0,0 +1,333 @@
+
+An interesting result is that many people seem to do not use backfilling as
+there’s no other way than to do it by CLI.
+
+**What executor type do you use?**
+
+|          |No.|  %   |
+|----------|---|------|
+|Celery    |138|44.81%|
+|Local     | 85|27.60%|
+|Kubernetes| 52|16.88%|
+|Sequential| 22|7.14% |
+|Other     | 11|3.57  |
+
+The other option mostly consisted of information that someone uses few types or is
+migrating from one executor to another. What can be observed is an increase in usage
+of Local and Kubernetes executors when compared to results from earlier [survey done
+by Ash](https://ash.berlintaylor.com/writings/2019/02/airflow-user-survey-2019/).
+
+**Do you use Kubernetes-based deployments for Airflow?**
+
+|                                                                     |No.|  %   |
+|---------------------------------------------------------------------|---|------|
+|No - we do not plan to use Kubernetes near term                      | 88|28.57%|
+|Yes - setup on our own via Helm Chart or similar                     | 65|21.10%|
+|Not yet - but we use Kubernetes in our organization and we could move| 61|19.81%|
+|Yes - via managed service in the cloud (Composer / Astronomer etc.)  | 45|14.61%|
+|Not yet - but we plan to deploy Kubernetes in our organization soon  | 42|13.64%|
+|Other                                                                |  7|2.27% |
+
+The most interesting thing is that there’s nearly 30% of users who do not use Kubernetes
+and they are not going to move. This means we should keep Kubernetes in mind when working
+on Airflow 2.0. On the other hand, almost 70% of the users either already use Kubernetes
+or it’s a viable option for them.
+
+**Do you combine multiple DAGs?**
+
+|                                 |No.|  %   |
+|---------------------------------|---|------|
+|No, I don't combine multiple DAGs|127|41.23%|
+|Yes, through SubDAG              | 73|23.70%|
+|Yes, by triggering another DAG   | 72|23.38%|
+|Other                            | 36|11.69%|
+
+In the other category, there were 9 people who explicitly mentioned using
+`ExternalTaskSensor` and I think it could be treated as running subDAGs by
+triggering other DAGs.
+
+**Do you use Airflow Plugins? If yes, what do you use it for?**
+
+|                                      |No.|  %   |
+|--------------------------------------|---|------|
+|Adding new operators/sensors and hooks|187|60.71%|
+|I don't use Airflow plugins           |109|35.39%|
+|Adding AppBuilder views & menu items  | 31|10.06%|
+|Adding new executor                   | 18|5.84% |
+|Adding OperatorExtraLinks             |  7|2.27% |
+
+The high percentage - 60%  for “Adding new operators/sensors and hooks” is quite a
+surprising result for some of us - especially that you do not actually need to use
+plugin mechanism to add any of those. Those are standard python objects and you can
+simply drop your hooks/operators/sensors code to PYTHONPATH and they will work.
```

Comment:
Agreed with you, although there is no guide explaining the best practice I think.
This might also come from the great https://github.com/airflow-plugins which (even though some of the repos are dated) seem to indicate new comers that the way to extend your DAG repo is by adding plugins.

Reply:
I will add your point